.github/release-please.yml

```diff
@@ -0,0 +1,3 @@
+releaseType: java-yoshi
```

Comment:
Is this for yoshi automation?

Reply:
Yes, this is for the Yoshi ReleasePlease bot.